### PR TITLE
Added region tags for shared VPC host and service projects

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**
Added region tags for shared VPC host and service projects
For b/184973701
Snippets to be included on https://cloud.google.com/vpc/docs/provisioning-shared-vpc

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: examples/shared_vpc/main.tf [hunk 1]
```diff
@@ -41,15 +41,18 @@ provider "random" {
 /******************************************
   Host Project Creation
  *****************************************/
+# [START vpc_shared_vpc_host_project_create]
 module "host-project" {
-  source                         = "../../"
+  source                         = "terraform-google-modules/terraform-google-project-factory/google//modules/shared_vpc_access"
```
Comment (morgante) [inline examples/shared_vpc/main.tf:46]: This isn't the right source. What are you trying to show with this example?

**Review by morgante — CHANGES_REQUESTED** [review]

File: examples/shared_vpc/main.tf (hunk 1 above)
Comment (betsy-lichtenberg) [reply examples/shared_vpc/main.tf:46]: I'm trying to add some Terraform tabs to this page: https://cloud.google.com/vpc/docs/provisioning-shared-vpc#setting_up

Comment (morgante) [reply examples/shared_vpc/main.tf:46]: Got it. This isn't the right example for that, as this actually *creates* the host project. Is that what you want to do?

If you just want to activate Shared VPC on an existing project, a new example would be needed.

Comment (betsy-lichtenberg) [reply examples/shared_vpc/main.tf:46]: Got it. Thanks for the guidance. I'm closing this PR and starting over with a new example.

**Pull request closed without merging**